registration/views/common.py

```diff
@@ -69,7 +71,7 @@ def getOptionsDict(orderItems):
                 ao.optionValue == 0
                 or ao.optionValue is None
                 or ao.optionValue == ""
-                or ao.optionValue == False
+                or ao.optionValue is False
```

Comment:
`or ao.optionValue == False`

or

`or not ao.optionValue`

Reply:
cant fix wont fix E712